core/src/main/kotlin/configuration.kt

```diff
@@ -24,6 +25,9 @@ object DokkaDefaults {
     const val noJdkLink: Boolean = false
     val analysisPlatform: Platform = Platform.DEFAULT
     const val suppress: Boolean = false
+
+    const val displayName = "JVM"
```

Comment:
What about calling this property "sourceSetDisplayName". I know this is not the very name of the SourceSet's property, but I think this would clarify it a little bit better 🙄

Reply:
Done